internal/pkg/connectors/modulr/client/transactions.go

```diff
@@ -6,6 +6,7 @@ import (
 	"net/http"
 )
 
+//nolint:tagliatelle // allow different styled tags in client
```

Comment:
I don't get what tagliatelle is enforcing here? It seems correct camelCase to me?